compiler/rustc_codegen_llvm/Cargo.toml

```diff
@@ -12,7 +12,10 @@ bitflags = "2.4.1"
 itertools = "0.12"
 libc = "0.2"
 measureme = "11"
-object = { version = "0.36.2", default-features = false, features = ["std", "read"] }
+object = { version = "0.36.2", default-features = false, features = [
+    "std",
+    "read",
+] }
```

Comment:
Maybe revert the rest of the changes to this file too?

Reply:
Sure. That's my IDE's auto formatter having fun again.